htmleditor-mixin.js

```diff
@@ -739,17 +740,53 @@ export const HtmlEditorMixin = superclass => class extends Localizer(LabelledMix
 					editor.undoManager.clear();
 				});
 
-				const persistEditorContents = () => {
-					if (this.pasteLocalImages) editor.uploadImages();
+				const persistEditorContents = async() => {
+
+					if (this._context?.useNewTinyMCEVersion) {
```

Comment:
```suggestion
					if (this._context && this._context.useNewTinyMCEVersion) {
```
Unfortunately null chaining isn't supported in all of our maintenance browsers yet.

Reply:
Though actually, this does technically bleed out of the feature flag, since this call wasn't async before. Even if there's nothing being awaited in an async method, the browser still schedules its run-time differently than if it wasn't async. So I think the flag check should control two versions of this method and the handlers below; flag off === synchronous and the original code, and flag on === async and the new code.

Reply:
I made some changes to the way things are flagged. I'm unsure if this is the best way to go about flagging since the methods can be synchronous vs asynchronous, but it made sense to me.